### PR TITLE
60086_updating_ui

### PR DESCRIPTION
**Pull request by ahammad-reyad**
Updated UI

### CODE REVIEW

**Comment by ahammad-reyad** [comment]
Updated UI

File: app/views/static_pages/home.html.erb [hunk 1]
```diff
@@ -8,14 +8,18 @@
...
-      <ul class="users">
-        <%= render @activities %>
-      </ul>
-      <%= will_paginate @activities %>
-    </div>
+    <aside class="col-md-8">
+      <section>
+        <%= link_to t(:all_categories), categories_path, class: "btn btn-lg btn-primary" %>
+        <%= link_to t(:all_words), words_path, class: "btn btn-lg btn-primary" %>
+     </section>
```
Comment (dieunb) [inline app/views/static_pages/home.html.erb]: indent

File: app/views/static_pages/home.html.erb [hunk 2]
```diff
@@ -8,14 +8,18 @@
...
+     <section>
```
Comment (dieunb) [inline app/views/static_pages/home.html.erb]: indent

File: app/views/static_pages/home.html.erb [hunk 3]
```diff
@@ -8,14 +8,18 @@
...
+        <ul class="users">
+          <%= render @activities %>
+        </ul>
+        <%= will_paginate @activities %>
+    </section>
```
Comment (dieunb) [inline app/views/static_pages/home.html.erb]: indent

**Pull request merged**